zeebe/restore/pom.xml

```diff
@@ -28,6 +28,11 @@
       <artifactId>zeebe-backup</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-db</artifactId>
```

Comment:
:thought_balloon: Instead of adding a dependency on `zeebe-db` to make `ChecksumProviderRocksDBImpl available, we could also promote the `TestChecksumProvider` to a `NoopChecksumProvider` and use that instead. We don't actually need the provider during restore so I think I'd favor this.